dotCMS/src/main/java/com/dotcms/rest/api/v1/site/SiteResource.java

```diff
@@ -31,8 +58,12 @@
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
+
+import io.vavr.control.Try;
 import org.apache.commons.lang.StringUtils;
 import org.glassfish.jersey.server.JSONP;
+import org.quartz.SchedulerException;
+import org.quartz.SimpleTrigger;
 
 /**
  * This resource provides all the different end-points associated to information
```

Comment:
Are you sure this is the right Resource to put this? since according to this javadoc this Resource is meant to the Site Browser?

Reply:
the doc was wrong